spec/rubocop/cop/rspec/unspecified_exception_spec.rb

```diff
@@ -254,5 +254,27 @@ def raise_error
         }.to raise_exception(StandardError)
       RUBY
     end
+
+    it 'skips ancestors that are not send types' do
+      node = parse_source(<<~RUBY).ast
+        expect {
+          raise StandardError
+        }.to raise_error
+      RUBY
+
+      allow_any_instance_of(RuboCop::AST::Node)
```

Comment:
I think I would rather have missing code coverage than using `allow_any_instance_of` in our specs 😅 Let’s see if we can find another approach.

Reply:
That's fair. 

I'm going to close this PR, I may or may not circle back to this one as I don't have any other ideas on how to test this right now.